pkg/mdformatter/mdgen/mdgen.go

```diff
@@ -40,19 +37,22 @@ func (t *genCodeBlockTransformer) TransformCodeBlock(ctx mdformatter.SourceConte
 	}
 	infoStringAttr := map[string]string{}
 	for i, field := range infoFiels {
-		val := strings.Split(field, "=")
+		val := []string{field}
+		if i := strings.Index(field, "="); i != -1 {
+			val = []string{field[:i], field[i+1:]}
```

Comment:
Awesome! Now it is only split at first "=". But maybe instead of using `Index` this can be done with something like,
```
val := strings.SplitN(field, "=", 2)
```
This also would only split at first "=" and return slice of length 2. WDYT?🙂

Reply:
Tried to use it, but found it confusing why 2 not 1.. (: will try thanks!

Reply:
I think 2 is the number of substrings returned...but yes it is not intuitive. This can be skipped then I guess.